### PR TITLE
Update github workflows

### PR DESCRIPTION
**Pull request by jenniferboedker**
Add missing workflow files, already fixed the names as well

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -39,10 +38,10 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v2
-    - name: Set up JDK 1.8
+    - name: Set up JDK 1.9
```
Comment (KochTobi) [inline .github/workflows/codeql-analysis.yml:41]: Is this the only repo where we use Java 9?

File: .github/workflows/create-release.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+      - name: Open PR with version bump
+        uses: actions/github-script@v4.0.2
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            await github.request(`POST /repos/${{ github.repository }}/pulls`, {
+              title: 'Update version to ${{ github.event.inputs.versionTag }}',
+              head: 'release/set-version-to-${{ github.event.inputs.versionTag }}',
+              base: 'main'
```
Comment (KochTobi) [inline .github/workflows/create-release.yml:88]:
```suggestion
              base: 'master'
```

**Review by KochTobi — CHANGES_REQUESTED** [review]

File: .github/workflows/create-release.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      versionTag:
+        description: 'Version Tag (semantic version)'
+        required: true
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.9
```
[the remark this answers is not in the document]
Comment (jenniferboedker) [reply .github/workflows/create-release.yml:15]: The other workflows were set to java 9, but we can also leave this PR as draft for @luiskuhn

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (jenniferboedker) [reply .github/workflows/codeql-analysis.yml:41]: see my other comment

**Review by KochTobi — APPROVED** [review]

**Pull request merged**